docs/troubleshooting.asciidoc

```diff
@@ -90,3 +90,24 @@ For this you'll need an `IApmLogger` implementation (see above) which you need t
 ----
 Agent.Setup(new AgentComponents(logger: new ApmLoggerBridge()));
 ----
+
+[float]
+[[double-agent-initialization]]
+=== An `InstanceAlreadyCreatedException` exception is thrown
+
+It can happen that in a very early stage of the monitored process the agent throws an `InstanceAlreadyCreatedException` exception with the following message: "The singleton APM agent has already been instantiated and can no longer be configured."
+
+This happens when the agent would be initialized multiple times, which is prohibited.
```

Comment:
```suggestion
It can happen that in a very early stage of the monitored process the agent throws an `InstanceAlreadyCreatedException` exception with the following message: "The singleton APM agent has already been instantiated and can no longer be configured." This happens when the agent would be initialized multiple times, which is prohibited.
```